toolchain/binutils-2.43-04-aarch64-phoenix.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/bfd/config.bfd b/bfd/config.bfd
+index bc65a681..121302fe 100644
+--- a/bfd/config.bfd
++++ b/bfd/config.bfd
+@@ -294,6 +294,11 @@ case "${targ}" in
+     targ_selvecs="aarch64_elf64_le_vec aarch64_elf32_le_vec aarch64_elf32_be_vec arm_elf32_be_vec arm_elf32_le_vec"
+     want64=true
+     ;;
++  aarch64-*-phoenix*)
++    targ_defvec=aarch64_elf64_le_vec
++    targ_selvecs="aarch64_elf64_be_vec arm_elf32_le_vec arm_elf32_be_vec"
++    want64=true
++    ;;
+   alpha*-*-freebsd* | alpha*-*-kfreebsd*-gnu)
+     targ_defvec=alpha_elf64_fbsd_vec
+     targ_selvecs="alpha_elf64_vec alpha_ecoff_le_vec"
+diff --git a/gas/configure.tgt b/gas/configure.tgt
+index ba42b2c8..37764241 100644
+--- a/gas/configure.tgt
++++ b/gas/configure.tgt
+@@ -139,6 +139,7 @@ case ${generic_target} in
+   aarch64*-*-nto*)			fmt=elf;;
+   aarch64*-*-openbsd*)			fmt=elf;;
+   aarch64*-*-pe* | aarch64*-*-mingw*)	fmt=coff em=pepaarch64 ;;
++  aarch64*-*-phoenix*)			fmt=elf;;
+   alpha-*-*vms*)			fmt=evax ;;
+   alpha-*-osf*)				fmt=ecoff ;;
+   alpha-*-linux*ecoff*)			fmt=ecoff ;;
+diff --git a/ld/Makefile.am b/ld/Makefile.am
+index f4358815..6e27457b 100644
+--- a/ld/Makefile.am
++++ b/ld/Makefile.am
+@@ -394,6 +394,7 @@ ALL_64_EMULATION_SOURCES = \
+ 	eaarch64linuxb.c \
+ 	eaarch64nto.c \
+ 	eaarch64pe.c \
++	eaarch64phoenix.c \
```

Comment:
This entry as well as one in `ld/Makefile.in` are redundant.

THey are only needed if we provide a new linkers configuration (`.sh` files, eg `ld/emulparams/elf32_sparc_phoenix.sh`).

On other platforms we provide those configs even though often times they are redundant. IMHO using the standard config is good enough for now.